bin/dev_setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Install components, setup DB and Solr:
+bundle install
+bundle exec rake argo:install
+
+# Run the server
+bundle exec rake jetty:stop  # Ensure it's not running
+bundle exec rake jetty:start # This may take a few minutes
+
+# Load and index records in the 'development' Solr core
+bundle exec rake argo:repo:load
```

Comment:
it looks to me like the `dev_setup` code, without the `bundle install`, could be made into an `argo:dev:setup` rake task. you'd then be able to use ruby code to test whether jetty has booted and fedora/solr are responding before the `argo:repo:load`.